tests/src/JIT/Regression/JitBlue/vso_176584/vso_176584.cs

```diff
@@ -0,0 +1,35 @@
+static class MicrosftNETBug
```

Comment:
Make sure you run this through the codeformatter, which I believe will also add the standard comment header.